tests/installation/boot_into_snapshot.pm

```diff
@@ -23,10 +23,9 @@ use base "opensusebasetest";
 
 sub run {
     my ($self) = @_;
-    assert_screen 'linux-login', 200;
```

Comment:
This is done to avoid race conditions. I fear this will cause a lot of failures on production.

I assume this line was removed because it caused a failure in a verification run. Could you paste the link to it?

Reply:
hm. I don't understand rice condition here. Can you explain why we can run into failure without checking linux-login?
I think select_console 'root-console'; should ensure console comes up. 
Please see provided verification tests run above.